modules/apps/account/account-test/src/testIntegration/java/com/liferay/account/service/test/AccountGroupLocalServiceTest.java

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.account.service.test;
+
+import com.liferay.account.model.AccountGroup;
+import com.liferay.account.service.AccountGroupLocalService;
+import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
+import com.liferay.portal.kernel.test.util.RandomTestUtil;
+import com.liferay.portal.test.rule.Inject;
+import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Albert Lee
+ */
+@RunWith(Arquillian.class)
+public class AccountGroupLocalServiceTest {
+
+	@ClassRule
+	@Rule
+	public static final LiferayIntegrationTestRule liferayIntegrationTestRule =
+		new LiferayIntegrationTestRule();
+
+	@After
+	public void tearDown() {
+		for (AccountGroup accountGroup : _accountGroups) {
+			accountGroup = _accountGroupLocalService.fetchAccountGroup(
+				accountGroup.getAccountGroupId());
+
+			if (accountGroup != null) {
+				_accountGroupLocalService.deleteAccountGroup(accountGroup);
+			}
+		}
+	}
```

Comment:
@alee8888 if you haven't done this already, check to see if we can apply the method-scoped `DataGuard` annotation to this class instead of doing manual cleanup. I believe it will also account for the delete test cases as well.

Reply:
@drewbrokke I have not. Will do, thanks!